source/snes9xgx.h

```diff
@@ -63,6 +63,24 @@ enum
 const char ctrlName[6][24] =
 { "SNES Controller", "SNES Mouse", "Super Scope", "Justifier", "SNES Controllers (2)", "SNES Controllers (4)" };
 
+enum {
+	TURBO_BUTTON_RSTICK = 0,
+	TURBO_BUTTON_A,
```

Comment:
it's minor, but I really don't see someone wanting to map turbo to the A button for example. Shouldn't we just simplify this by removing the buttons no-one would ever use from this list?

Reply:
You're right in that it's probably unlikely - but given the fact that the SNES buttons are remappable to any button on the input controller, I felt obliged to account for that flexibility as much as possible.

For example, if someone had remapped one or more face buttons to triggers, and wanted to use a face button as Turbo (like A), that option is there for them.